extensions/default/src/Panels/StudyBrowser/PanelStudyBrowser.tsx

```diff
@@ -356,10 +346,51 @@ function PanelStudyBrowser({
         showSettings={actionIcons.find(icon => icon.id === 'settings').value}
         viewPresets={viewPresets}
         ThumbnailMenuItems={props => (
-          <CustomizedThumbnailMenuItems
-            {...props}
-            commandsManager={commandsManager}
-          />
+          <DropdownMenu>
```

Comment:
Can this be externalized as a shared class?  It feels like 
<CustomizedDropdownMenu customizationService={customizationService} menuId="studyBrowser.thumbnailMenuItems" />
would allow both menus to use the same component.